### PR TITLE
Added FutaFilter List

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/FiltersRegistry/issues/430

### CODE REVIEW

**Review by zloyden — APPROVED** [review]

File: filters/ThirdParty/filter_250_FutaFilter/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 250,
+  "name": "FutaFilter List",
+  "description": "An advertisement blocking list specialized for Taiwanese users.",
+  "timeAdded": 1616091726943,
+  "homepage": "https://github.com/FutaGuard/FutaFilter/blob/master/README.en.md",
```
Comment (ameshkov) [inline filters/ThirdParty/filter_250_FutaFilter/metadata.json:6]: For regional list we can use the default homepage.

File: filters/ThirdParty/filter_250_FutaFilter/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 7,
+  "subscriptionUrl": "https://filter.futa.gg/filter.txt",
+  "tags": [
+    "purpose:ads",
+    "reference:101",
+    "lang:zh"
```
Comment (ameshkov) [inline filters/ThirdParty/filter_250_FutaFilter/metadata.json:14]: Can't we use `zh-TW` here?

**Review by ameshkov — CHANGES_REQUESTED** [review]
Let's wait a little bit, see the discussion here: https://github.com/AdguardTeam/FiltersRegistry/issues/430#issuecomment-802529520

File: filters/ThirdParty/filter_250_FutaFilter/metadata.json (hunk 2 above)
Comment (slavaleleka) [reply filters/ThirdParty/filter_250_FutaFilter/metadata.json:14]: probably we should add it to `tags/metadata.js` first

**Comment by zloyden** [comment]
May be reopened if the author continues to support the filter.

**Pull request closed without merging**